src/utils/tests/ProviderWrapper.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactNode } from 'react';
+import { Provider as StoreProvider } from 'react-redux';
+import { AuthProvider } from '../../context/AuthContext';
+import { store } from '../../../App';
+
+interface ProviderWrapperProps {
+  children: ReactNode;
+}
+
+const ProviderWrapper: React.FC<ProviderWrapperProps> = ({ children }) => (
+  <StoreProvider store={store}>
+    <AuthProvider>
+      <>{children}</>
```

Comment:
Why this need to wrap in fragment?

Reply:
why we are allows AuthProvider to be  null or undefined as children

Reply:
I don't remember exactly why I did this, removed.
https://github.com/Real-Dev-Squad/mobile-app/pull/477/files#diff-57cdfd62a0137f975515ef87ba1e94e78cc9fa0bfc6d1f215e3105896b4c1fccR11-R14